### DISCUSSION TITLE
Lora tag in prompt

### DISCUSSION THREAD

**Discussion by drphero** [glibsonoran/Plush-for-ComfyUI#52] [General]
I noticed that in the "Advanced Prompt Enhancer and Dalle_e" example workflow that there is <lora:MyLora:1> in the prompt box. Does this actually work? I didn't think comfyui supported loras being loading from the prompt.

**Comment by glibsonoran** [comment]
Yes, although I think that was more common when using 1.5 models than with SDXL. Of course I just put it in there as an example, I think most Comfy users just use lora loader nodes.
